.github/workflows/go-tests.yml

```diff
@@ -446,17 +449,19 @@ jobs:
       consul-license: ${{secrets.CONSUL_LICENSE}}
       datadog-api-key: "${{ !endsWith(github.repository, '-enterprise') && secrets.DATADOG_API_KEY || '' }}"
 
-  go-test-sdk-1-19:
+  go-test-sdk-backwards-compatibility:
```

Comment:
I love this. No more twiddling versions whenever we update the go major version.